src/aat/resources/features/F-102/F-102.feature

```diff
@@ -50,3 +50,13 @@ Feature: F-102: Get jurisdictions available to the user
     And the response [contains HTTP 400 Bad Request]
     And the response [contains an error message : Access can only be 'create', 'read' or 'update']
     And the response has all other details as expected
+
+  @S-580
+  Scenario: must return a list of jurisdictions for a valid user with no user profile
```

Comment:
must return a list of jurisdictions for a valid user with no **CCD** user profile